### DISCUSSION TITLE
helmfile generates output file with wrong indentation - 4 spaces for tab

### DISCUSSION THREAD

**Discussion by sf-riqbal** [helmfile/helmfile#752] [Questions] [answered]
Running helmfile to generate output file changes the indentation on some machines. Indentation is changed from 2 spaces for a tab to 4 spaces. This results in different team members ending up generating different format of the file. 
Looks like it is using sme local machine configuration but I have verified both Vim and VsCode editors and they are both set to 2 spaces for a tab. Is there any specific configuration that helmfile is using?

I am on Mac and this is the command I use
helmfile --namespace ${project} -l name=${name}-e ${env} -f source/helmfile.yaml write-values --output-file-template "$(pwd)/${dest_dir}/kustomization/values/values.yaml"

**Comment by yxxhero** [comment]
@sf-riqbal what's your expected result? and which helmfile version?

**Comment by yxxhero** [comment]
maybe we can make it configurable.

**Reply by sf-vorlov** [reply]
@yxxhero I'll answer for @sf-riqbal. 
v0.150.0
We'd expect to achieve consistency for all the users.
By the way, we only see this behaviour with the write-values, but not the template.

**Comment by sf-riqbal** [comment]
@yxxhero Thanks for looking into it. the version I was using was v0.149.0 which I have now upgraded to v0.151.0 and the issue is resolved for me. I will confirm with other team members to make sure issue was with older version and close this case.

**Comment by sf-riqbal** [comment, marked as answer]
Closing this case as I can confirm that issue was related to v0.149.0 version

**Comment by yxxhero** [comment]
@sf-riqbal Thanks for your feedback.
